src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2385,25 +2385,25 @@
     <value>Preview popup</value>
   </data>
   <data name="ToggleCompactOverlay" xml:space="preserve">
-    <value>Toggle compact overlay</value>
+    <value>Compact overlay</value>
   </data>
   <data name="OpenHelpDescription" xml:space="preserve">
     <value>Open the online help page in your browser</value>
   </data>
   <data name="ToggleFullScreenDescription" xml:space="preserve">
-    <value>Toggle full screen mode</value>
+    <value>Toggle between full screen mode</value>
   </data>
   <data name="EnterCompactOverlayDescription" xml:space="preserve">
-    <value>Enter compact overlay mode</value>
+    <value>Switch to compact overlay mode</value>
   </data>
   <data name="ExitCompactOverlayDescription" xml:space="preserve">
-    <value>Exit compact overlay mode</value>
+    <value>Switch to normal overlay mode</value>
```

Comment:
The app is either in "compact overlay mode' or not, 'normal overlay' isn't a mode.

Reply:
It doesn't really have a name, the window is either in the Compact Overlay state or not. The closest I can think of is 'Regular mode', but that doesn't really provide a lot of context.